go/vt/vtadmin/api.go

```diff
@@ -962,3 +1027,25 @@ func (api *API) VTExplain(ctx context.Context, req *vtadminpb.VTExplainRequest)
 		Response: response,
 	}, nil
 }
+
+func (api *API) getClustersForRequest(ids []string) ([]*cluster.Cluster, []string) {
```

Comment:
just moving this to the bottom of the file; it felt a bit out of place being stuck between two public functions without really being a helper function for either of them in particular